### PR TITLE
Hunt exclusions V

### PR DESCRIPTION
**Pull request by sudiksha27**
:(

**Priority (whenever / two weeks / yesterday)**:
Before end of week is ideal

### CODE REVIEW

File: leader-election-impl/src/main/java/com/palantir/paxos/PaxosStateLogImpl.java [hunk 1]
```diff
@@ -56,18 +55,16 @@
...
+    private static boolean checkNameIsALong(File file) {
+        if (file == null) {
+            return false;
+        }
+        try {
+            getSeqFromFilename(file);
+            return true;
+        } catch (NumberFormatException e) {
+            return false;
+        }
```
Comment (jeremyk-91) [inline leader-election-impl/src/main/java/com/palantir/paxos/PaxosStateLogImpl.java:67]: I realise the transform is mechanical, and this is correct, but this still gives me chills in a bad way 😅

File: timelock-server/src/testCommon/java/com/palantir/atlasdb/timelock/TimeLockServerHolder.java [hunk 2]
```diff
@@ -168,4 +159,13 @@ TimeLockInstallConfiguration installConfig() {
             throw new RuntimeException(e);
         }
     }
+
+    private static WireMockConfiguration wireMockConfiguration(int port) {
```
Comment (jeremyk-91) [inline timelock-server/src/testCommon/java/com/palantir/atlasdb/timelock/TimeLockServerHolder.java:163]: Here's a fun one: I think this should take an `Integer` as opposed to an `int` (why?)

**Review by jeremyk-91 — APPROVED** [review]
👍

File: timelock-server/src/testCommon/java/com/palantir/atlasdb/timelock/TimeLockServerHolder.java (hunk 2 above)
Comment (sudiksha27) [reply timelock-server/src/testCommon/java/com/palantir/atlasdb/timelock/TimeLockServerHolder.java:163]: because httpsport takes an `Integer`?

**Review by jeremyk-91 — APPROVED** [review]

**Pull request merged**